src/lib/apollo/client.ts

```diff
@@ -26,9 +36,31 @@ const subgraphUri = () => {
   return url.href
 }
 
+const typesMap: FunctionsMap = {
```

Comment:
while the changes in the codegen.yml file allow _typing_ BigNumbers, these changes enable _parsing_ the values as BigNumbers